.github/pull_request_template.md

```diff
@@ -0,0 +1,84 @@
+<!--
+You've got a Pull Request you want to submit? Awesome!
+This PR template is here to help ensure you're setup for success:
+  please fill it out to help ensure that your PR is complete and ready for approval.
+-->
+
+**JIRA Ticket:**
+<https://jira.cms.gov/browse/xxxx>
+
+**User Story or Bug Summary:**
+<!-- Please copy-paste the brief user story or bug description that this PR is intended to address. -->
+
+
+### What Does This PR Do?
+
+<!--
+Add detailed discussion of changes here
+This is likely a summary, or the complete contents, of your commit messages.
+-->
+
+
+### What Should Reviewers Watch For?
+
+If you're reviewing this PR, please check these things, in particular:
+
+* TODO
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply for some reason.
+
+Common items include:
+* Is this likely to address the goals expressed in the user story?
+* Are any additional documentation updates needed?
+* Are there any unhandled and/or untested edge cases you can think of?
+* Is user input properly sanitized & handled?
+* Does this make any backwards-incompatible changes that might break end user clients?
+* Can you find any bugs if you run the code locally and test it manually?
+-->
+
+
+### What Security Implications Does This PR Have?
+
+Submitters should complete the following questionnaire:
+
+* If the answer to any of these questions is **Yes**, then here's a link to the associated Security Impact Assessment: N/A.
+    * Does this PR add any new software dependencies? **Yes** or **No**.
+    * Does this PR modify or invalidate any of our security controls? **Yes** or **No**.
+    * Does this PR store or transmit data that was not stored or transmitted before? **Yes** or **No**.
+* If the answer to any of these questions is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
```

Comment:
This seems like a pretty heavy burden on @StewGoin.  Not to mention, is it a blocker if @StewGoin is out or on vacation?
BCDA has a security checklist process which we go through _before_ doing any work, and get it approved by @StewGoin.  Is this new process intended to deprecate that?

Reply:
I think you're reading it backwards? Jason only needs to get tagged for a "Yes" to the question _below_ that.

Or am I misunderstanding you?

Reply:
Ahhh, yes I think you are right.  If I am _now_ understanding you correctly, Jason only gets tagged for this question:
```Do you think this PR requires additional review of its security implications for other reasons?```

I definitely thought it applied to the above section, likely due to the raw format of the file under review.  I suspect, that once properly formatted with markdown, there will be no confusion here.  

Thanks for clarifying.

Reply:
Any concerns with that, then? I think it's prompting folks to ask themselves, "_should_ I maybe talk to Jason about this first?" Which is a good thing to think about!

And yeah, if he's out, then... 🤷. It can likely wait. And if it can't, then folks are smart enough to ask for help, I think.

Reply:
Totally agree.  I've seen lots of folks do this naturally, but having the prompt in the template is definitely worthwhile.